src/main/java/net/openhft/chronicle/wire/Wires.java

```diff
@@ -77,8 +77,9 @@ public enum Wires {
     public static final int SPB_HEADER_SIZE = 4;
     public static final List<Function<Class, SerializationStrategy>> CLASS_STRATEGY_FUNCTIONS = new CopyOnWriteArrayList<>();
 
-    @Deprecated(/* for removal in x.26 */)
+    // @Deprecated(/* for removal in x.26 */)
```

Comment:
If de-deprecating let's delete or bump deprecation to future version

Reply:
Added a comment as to why this is commented out. Not de-deprecating it